data/korath/korath ships.txt

```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		"Langrage Hyper-Heaver" 1
+		"Firelight Missile Bank"
+		"Firelight Missile" 18
+
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 2
+		"Fuel Processor" 2
+		"Korath Repeater Rifle" 523
+		"Microbot Defense Station" 18
+
+		"Afterburner (Planetary Class)"
+		"Reverser (Lunar Class)"
+		"Thruster (Planetary Class)"
+		"Steering (Planetary Class)"
+		"Scram Drive"
+		"Jump Drive"
+
+	"reverse engine" -79 -186
+		zoom 0.8
+	"reverse engine" 79 -186
+		zoom 0.8
+	engine -101 196
+		zoom 0.9
+	engine 101 196
+		zoom 0.9
+	engine -60 190 0.8
+	engine 60 190 0.8
+	engine -29 212
+	engine 29 212
+	gun -81.5 58 "Firelight Missile Bank"
+		angle 270
+	turret -89 -134 "Banisher Grav-Turret"
+	turret 89 -134 "Warder Anti-Missile"
+	turret -110 -126 "Warder Anti-Missile"
+	turret 110 -126 "Banisher Grav-Turret"
+	turret 36.5 -55.5 "Langrage Hyper-Heaver"
+	turret -63.5 44 "Warder Anti-Missile"
+	turret 63.5 44 "Banisher Grav-Turret"
+	turret -63.5 66.5 "Banisher Grav-Turret"
+	turret 63.5 66.5 "Warder Anti-Missile"
+	bay "Fighter" -115 -152
+		angle 30
+	bay "Fighter" 115 -152
+		angle 330
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "Although fierce in battle and highly adaptable, Korath do not easily recover from injury. Without the aid of a surgeon or more equipment than fits into a standard escape pod, a stranded, injured Korath faces poor odds of survival. Therefore, Korath who leave their home territory have placed great importance on the construction of hospital ships that can quickly locate, recover, and treat the survivors of wrecked ships."
+	description "	The Rai'alorej-class Refuge, named for a pioneer in the field of microbot-assisted surgery, is thoroughly appointed with lifeform scanners, locator beacon receivers, ambulances, and search-and-rescue teams. Once onboard, crash victims are greeted with the finest hospital facilities, microbot treatments, and stores of medicine the Exiles have at hand. Even incubators and nurseries are available to treat the young who grow up aboard ship."
+
+
+ship "Ikatila'ej"
+	sprite "ship/ikatila-ej"
+	thumbnail "thumbnail/ikatila-ej"
+	attributes
+		category "Utility"
+		"cost" 109375000
+		"shields" 72700
+		"hull" 93000
+		"required crew" 957
+		"bunks" 1933
+		"mass" 5380
+		"drag" 58
+		"heat dissipation" .29
+		"fuel capacity" 800
+		"cargo space" 845
+		"outfit space" 1829
+		"weapon capacity" 680
+		"engine capacity" 290
+		"spinal mount" 1
+		weapon
+			"blast radius" 600
+			"shield damage" 9000
+			"hull damage" 4800
+			"hit force" 20000
+	outfits
+		"Microbot Factory"
+		"Shunt-Strike Turret" 5
+		"Grab-Strike Turret" 4
+		"Warder Anti-Missile" 5
+
+		"Triple Plasma Core"
+		"Double Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 2
```

Comment:
```suggestion
```

Reply:
This is the wrong direction. We want less stuff built-in, not more.

Reply:
> This is the wrong direction. We want less stuff built in, not more.

This isn't quite true. 
Right now, we have some ships that have outfits built in, which aren't serving a purpose by being built-in. They aren't furthering lore by being built-in; they aren't providing a meaningful mechanical difference; they're built-in for convenience. 

Those are the kind of stats that we want to migrate out of ships and into outfits. 

However, in this case, this ship is entirely built for and designed around a single purpose. Therefore, having built-in stats like this is both lore appropriate and helps to differentiate this ship mechanically. 

A blanket statement of "built-in stats bad, don't have them" is remarkably limiting from a creative standpoint. We should make good use of all the available tools, not arbitrarily deciding that certain things are "bad" and should be avoided.

Reply:
A good example is the built-in cloak that Remnant ships use. It make sense that they're built in given the Remnant's history and culture; stealth is important to them. Making their cloak removable reduces their uniqueness and unbalances Remnant outfits. I'm not a stats guy so I don't know if built in outfits are a good idea in this case, but to make the blanket statement that built in outfits are bad, period, seems to be detrimental to the game.

Reply:
I agree, but I am also parroting Amazinite and others who have repeatedly told me that we should avoid building stuff into the hull if there is absolutely any way to accomplish the same thing using regular outfits.

Reply:
I'm not opposed to having stats built into the hull when it's something like a fighter that needs to squeeze out every tonne, or a Remnant ship whose cloaking is not something you could just outfit.

The idea of taking an existing outfit that the ship reasonably would have and building *that* into the hull, that is where it becomes a bad idea.

Why *wouldn't* a massive factory ship be able to have the space to swap out its reactors when one breaks down, or even lend one to someone who needs it?  This ship isn't starving for volume like a fighter, and the outfits that warp core is removing are not unique abilities.

One thing this is telling me, however, is that while I've been a bit back and forth as to whether or not this ship should be capturable, the fear that it might be stolen and used as a broken battleship tells me that it probably should not.  It's got enough microbots on board that it should be able to liquidate a KWS full of troops attempting to attack anyway, so the solution may just be to mark this uncapturable like the Korsmanath A'awoj.   Genocidal players may attempt to steal the outfits, perhaps even the grinder, but not the ship itself.

As much fun as it would be to mount a dragonflame on this and go on a rampage, such a ridiculous arrangement probably belongs to the realm of plugins.